frontend/server/src/DAO/CoderOfTheMonth.php

```diff
@@ -121,13 +121,12 @@ public static function getCandidatesToCoderOfTheMonth(
      * @return list<array{time: string, username: string, classname: string}>
      */
     final public static function getCodersOfTheMonthFromSchool(
-        int $schoolId,
-        string $category = 'all'
+        int $schoolId
     ): array {
         $date = date('Y-m-01', \OmegaUp\Time::get());
         $sql = "
             SELECT
-              cm.time,
+              distinct cm.time,
```

Comment:
```suggestion
            SELECT DISTINCT
              cm.time,
```

recuerda siempre intentar seguir el estilo. todas las palabras reservadas de SQL van en mayúsculas.